wrapper/src/main/java/software/amazon/jdbc/ConnectionPluginManager.java

```diff
@@ -259,19 +272,28 @@ protected <T, E extends Exception> PluginChainJdbcCallable<T, E> makePluginChain
       final Set<String> pluginSubscribedMethods = plugin.getSubscribedMethods();
       final String pluginName = pluginNameByClass.getOrDefault(plugin.getClass(), plugin.getClass().getSimpleName());
       final boolean isSubscribed =
-          pluginSubscribedMethods.contains(ALL_METHODS)
-              || pluginSubscribedMethods.contains(methodName);
+          (!useForceConnectPipeline || (plugin instanceof AuthenticationConnectionPlugin))
```

Comment:
if I understand correctly, only AuthenticationConnectionPlugins will potentially be subscribed to forceConnect. This means that for all non-auth plugins we can:
- remove the forceConnect method
- remove "forceConnect" from the subscribed methods if it is explicitly included
- move the implementation of connectInternal() connect() and then get rid of connectInternal()

It looks like this PR only removes forceConnect from AuroraInitialConnectionPlugin, can you update the PR with the above suggestions for non-auth plugins?